assets/js/phoenix_live_view/view.js

```diff
@@ -123,9 +124,13 @@ const serializeForm = (form, opts, onlyNames = []) => {
         onlyHiddenInputs[key] = true;
       }
 
+      const inputSkipUnusedField = input.hasAttribute(PHX_NO_USED_CHECK);
```

Comment:
To be consistent we should use `binding(PHX_NO_USED_CHECK)` to get the attribute name. I know it's not convenient since we'd need to pass it, but since we never use `serializeForm` elsewhere, I think we should just move serializeForm into the View class as a method instead, then replace calls to it with `this.serializeForm`. Then, we can just call `this.binding` inside.